From 2d0098977178fbf1b5e1d106c19c7e56debadd79 Mon Sep 17 00:00:00 2001
From: Haohui Mai <wheat9@apache.org>
Date: Wed, 18 Mar 2015 13:33:41 -0700
Subject: [PATCH 1312/2748] HDFS-7697. Mark the PB OIV tool as experimental.
 Contributed by Lei (Eddy) Xu.

(cherry picked from commit 572555c9f909e1bb7681030b1501f3317a53b7d8)

Conflicts:
	hadoop-hdfs-project/hadoop-hdfs/src/site/markdown/HdfsImageViewer.md

Change-Id: I5e34b4326eb4ccd2ae549f091a645902dc496229
---
 .../offlineImageViewer/OfflineImageViewerPB.java   |    6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/tools/offlineImageViewer/OfflineImageViewerPB.java b/hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/tools/offlineImageViewer/OfflineImageViewerPB.java
index dbcb786..fe05979 100644
--- a/hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/tools/offlineImageViewer/OfflineImageViewerPB.java
+++ b/hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/tools/offlineImageViewer/OfflineImageViewerPB.java
@@ -72,7 +72,7 @@
       + "    -step defines the granularity of the distribution. (2MB by default)\n"
       + "  * Web: Run a viewer to expose read-only WebHDFS API.\n"
       + "    -addr specifies the address to listen. (localhost:5978 by default)\n"
-      + "  * Delimited: Generate a text file with all of the elements common\n"
+      + "  * Delimited (experimental): Generate a text file with all of the elements common\n"
       + "    to both inodes and inodes-under-construction, separated by a\n"
       + "    delimiter. The default delimiter is \\t, though this may be\n"
       + "    changed via the -delimiter argument.\n"
@@ -87,10 +87,10 @@
       + "-p,--processor <arg>   Select which type of processor to apply\n"
       + "                       against image file. (XML|FileDistribution|Web|Delimited)\n"
       + "                       (Web by default)\n"
-      + "-delimiter <arg>       Delimiting string to use with Delimited processor\n"
+      + "-delimiter <arg>       Delimiting string to use with Delimited processor.  \n"
       + "-t,--temp <arg>        Use temporary dir to cache intermediate result to generate\n"
       + "                       Delimited outputs. If not set, Delimited processor constructs\n"
-      + "                       the namespace in memory before outputting text."
+      + "                       the namespace in memory before outputting text.\n"
       + "-h,--help              Display usage information and exit\n";
 
   /**
-- 
1.7.9.5

